allennlp/models/semantic_parsing/wikitables/wikitables_erm_semantic_parser.py

```diff
@@ -390,9 +371,14 @@ def _get_state_cost(self, worlds: List[WikiTablesWorld], state: CoverageState) -
         action_history = state.action_history[0]
         batch_index = state.batch_indices[0]
         action_strings = [state.possible_actions[batch_index][i][0] for i in action_history]
-        logical_form = world.get_logical_form(action_strings)
-        lisp_string = state.extras[batch_index]
-        if self._executor.evaluate_logical_form(logical_form, lisp_string):
+        logical_form = world.action_sequence_to_logical_form(action_strings)
```

Comment:
We can execute action sequences directly, which is definitely recommended here, so you don't have to go back and forth between strings.  It'll make things at least a little faster.

Reply:
Done. Thanks! Added a  method`WikiTablesLanguage.evaluate_action_sequence`